### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by FelixKla**
Pull Dev to main? Just added doxygen comments.

### CODE REVIEW

**Review by RelentlessRDS — APPROVED** [review]
Verwachting is wel dat Doxygen verder moet worden uitgebreid tussen nu en het einde van het project, maar dit is een solide basis

File: include/mpu6050.hpp
Comment (miep01) [inline include/mpu6050.hpp:1]: Please use correct namespace `asn`

File: src/motor_control.cpp
Comment (toob01) [inline src/motor_control.cpp:1]: is `direction_t` an enum? If so, please correct the casing to `Direction_t`

File: src/mpu6050.cpp
Comment (miep01) [inline src/mpu6050.cpp:1]: Here of course namespacing as well

**Review by miep01 — CHANGES_REQUESTED** [review]
For merging onto main, please correct small mistakes noted in review comments. Everything else is good.

**Review by miep01 — APPROVED** [review]
all good now

**Pull request merged**